project/Build.scala

```diff
@@ -495,79 +489,74 @@ object Build {
       testOptions += Tests.Argument(TestFrameworks.JUnit, "-v", "-a", "-s")
   )
 
-  lazy val irProject: Project = Project(
-      id = "ir",
-      base = file("ir"),
-      settings = commonIrProjectSettings ++ Seq(
-          libraryDependencies +=
-            "com.novocode" % "junit-interface" % "0.9" % "test"
-      )
+  lazy val irProject: Project = Project(id = "ir", base = file("ir")).settings(
+      commonIrProjectSettings,
+      libraryDependencies +=
+        "com.novocode" % "junit-interface" % "0.9" % "test"
   )
 
-  lazy val irProjectJS: Project = Project(
-      id = "irJS",
-      base = file("ir/.js"),
-      settings = commonIrProjectSettings ++ myScalaJSSettings ++ Seq(
-          crossVersion := ScalaJSCrossVersion.binary,
-          unmanagedSourceDirectories in Compile +=
-            (scalaSource in Compile in irProject).value,
-          unmanagedSourceDirectories in Test +=
-            (scalaSource in Test in irProject).value
-      )
+  lazy val irProjectJS: Project = Project(id = "irJS", base = file("ir/.js")
+  ).settings(
```

Comment:
There are a bunch of other occurrences of this pattern. I'll spare you the line comments everywhere ;)